packages/dialect-react-sdk/web2.api.ts

```diff
@@ -0,0 +1,254 @@
+// FIXME: this is a deprecated web2 api
```

Comment:
Do we still need this file or can it be safely removed?

Reply:
yes, we'll revisit it while refactoring notifications to new api(v1). currently it's just backported